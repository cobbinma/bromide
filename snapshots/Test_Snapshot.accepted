testStruct{example: "test", again: 5}