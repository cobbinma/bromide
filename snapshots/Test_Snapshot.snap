testStruct{example: "hello", again: 5}